third_party/move/move-model/src/model.rs

```diff
@@ -637,11 +644,21 @@ impl GlobalEnv {
         self.generated_by_v2 = yes
     }
 
-    /// Returns if this is generated by v2.
+    /// Returns true if this is generated by v2.
     pub fn generated_by_v2(&self) -> bool {
         self.generated_by_v2
     }
 
+    /// Sets the language version.
```

Comment:
Well, I used to do this 'redundant' comments anyway, to satisfy the principle that each public function has a comment. In my experience, if you don't do, people come along later and just add arbitrary undocumented functions, because they think its ok from other examples.